phys2bids/physio_obj.py

```diff
@@ -467,11 +468,15 @@ def check_trigger_amount(self, thr=None, num_timepoints_expected=0, tr=0):
                         'of time to find the starting time.')
             time = np.linspace(time[0], time[-1], len(trigger))
 
-        # Check if thr was given, if not "guess" it.
         flag = 0
         if thr is None:
-            thr = np.mean(trigger) + 2 * np.std(trigger)
-            flag = 1
+            thr = np.mean(trigger) + np.std(trigger)	            # If trigger channels are binary 
```

Comment:
If the threshold is gonna be the mean of the trigger, this line is not used at all then. Could you delete it?

Reply:
Oops! Yes, will do. that was a copy/paste error.